modules/shmelev_i_graham_algorithm/test/test_graham_algorithm.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright 2024 Shmelev Ivan
+
+#include <gtest/gtest.h>
+#include <unordered_set>
+#include <utility>
+#include <algorithm>
+
+#include "include/graham_algorithm.h"
+
+TEST(GrahamScanTest, TestEmptyVector) {
+    std::vector<std::pair<double, double>> points;
+    std::vector<std::pair<double, double>> result =
+                                 GrahamScan::convexHull(points);
+    EXPECT_TRUE(result.empty());
+}
+
+TEST(GrahamScanTest, TestOnePoint) {
+    std::vector<std::pair<double, double>> points = {{0, 0}};
```

Comment:
```suggestion
    std::vector<std::pair<double, double>> points{{0, 0}};
```
Также поправьте в других местах